<?xml version="1.0" encoding="UTF-8"?>
<robot generated="20181122 21:49:16.958" generator="Robot 3.0.2 (Python 2.7.15 on win32)">
<suite source="C:\D\plc_system_test" id="s1" name="Plc System Test">
<suite source="C:\D\plc_system_test\tc.4.robot" id="s1-s1" name="Tc.4">
<test id="s1-s1-t1" name="tc_4.1">
<kw name="Run Iot Test" library="plc_tb_ctrl.PlcSystemTestbench">
<arguments>
<arg>tc_4_1</arg>
</arguments>
<msg timestamp="20181122 21:49:17.165" level="INFO">./tc\tc_iot_4\tc_4_1.py</msg>
<msg timestamp="20181122 21:49:17.195" level="INFO">Open concentrator port</msg>
<msg timestamp="20181122 21:49:17.205" level="INFO">power off all of meters and cco, delay 5s</msg>
<msg timestamp="20181122 21:49:22.212" level="INFO">power on all of meters and cco.</msg>
<msg timestamp="20181122 21:49:22.212" level="INFO">Wait for gdw1376p2 frame. DIR=UL, AFN=3,DT1=2,DT2=1</msg>
<msg timestamp="20181122 21:49:25.686" level="INFO">Received</msg>
<msg timestamp="20181122 21:49:25.686" level="INFO">set CCO addr=00-00-00-00-00-9C</msg>
<msg timestamp="20181122 21:49:25.696" level="INFO">Wait for gdw1376p2 frame. DIR=UL, AFN=0,DT1=1,DT2=0</msg>
<msg timestamp="20181122 21:49:25.755" level="INFO">Received</msg>
<msg timestamp="20181122 21:49:25.755" level="INFO">reset CCO param area</msg>
<msg timestamp="20181122 21:49:25.755" level="INFO">Wait for gdw1376p2 frame. DIR=UL, AFN=0,DT1=1,DT2=0</msg>
<msg timestamp="20181122 21:49:25.815" level="INFO">Received</msg>
<msg timestamp="20181122 21:49:25.815" level="INFO">set sub node address to consorting cco, and start the escort net</msg>
<msg timestamp="20181122 21:49:25.815" level="INFO">set sub node address to main cco, and start the main net</msg>
<msg timestamp="20181122 21:49:25.815" level="FAIL">IOError: [Errno 2] No such file or directory: 'tc/tc_iot_4/\xe4\xba\x92\xe6\x93\x8d\xe4\xbd\x9c\xe6\x80\xa7\xe8\xa1\xa8\xe6\x9e\xb6\xe6\x8b\x93\xe6\x89\x91\xe5\x9c\xb0\xe5\x9d\x80_\xe5\xae\x8c\xe6\x95\xb4\xe7\x89\x88_\xe6\xa0\x91\xe5\xbd\xa2.txt'</msg>
<msg timestamp="20181122 21:49:25.815" level="DEBUG">Traceback (most recent call last):
  File "C:\D\plc_system_test\lib\plc_tb_ctrl.py", line 334, in run_iot_test
    tc.run(self)
  File "./tc\tc_iot_4\tc_4_1.py", line 73, in run
    f = open('tc/tc_iot_4/\xe4\xba\x92\xe6\x93\x8d\xe4\xbd\x9c\xe6\x80\xa7\xe8\xa1\xa8\xe6\x9e\xb6\xe6\x8b\x93\xe6\x89\x91\xe5\x9c\xb0\xe5\x9d\x80_\xe5\xae\x8c\xe6\x95\xb4\xe7\x89\x88_\xe6\xa0\x91\xe5\xbd\xa2.txt','r')</msg>
<status status="FAIL" endtime="20181122 21:49:25.815" starttime="20181122 21:49:17.145"></status>
</kw>
<kw type="teardown" name="Record Test Status" library="plc_tb_ctrl">
<msg timestamp="20181122 21:49:25.945" level="INFO">begin 5 seconds waiting before exit...</msg>
<msg timestamp="20181122 21:49:30.950" level="INFO">stop DUT logging</msg>
<msg timestamp="20181122 21:49:30.951" level="FAIL">AttributeError: 'PlcSystemTestbench' object has no attribute 'dut_log'</msg>
<msg timestamp="20181122 21:49:30.951" level="DEBUG">Traceback (most recent call last):
  File "C:\D\plc_system_test\lib\plc_tb_ctrl.py", line 166, in record_test_status
    TB_INSTANCE._stop_dut_logging()
  File "C:\D\plc_system_test\lib\plc_tb_ctrl.py", line 1020, in _stop_dut_logging
    terminate_process(self.dut_log)</msg>
<status status="FAIL" endtime="20181122 21:49:30.951" starttime="20181122 21:49:25.945">AttributeError: 'PlcSystemTestbench' object has no attribute 'dut_log'</status>
</kw>
<status status="FAIL" endtime="20181122 21:49:30.951" critical="yes" starttime="20181122 21:49:17.145">IOError: [Errno 2] No such file or directory: 'tc/tc_iot_4/\xe4\xba\x92\xe6\x93\x8d\xe4\xbd\x9c\xe6\x80\xa7\xe8\xa1\xa8\xe6\x9e\xb6\xe6\x8b\x93\xe6\x89\x91\xe5\x9c\xb0\xe5\x9d\x80_\xe5\xae\x8c\xe6\x95\xb4\xe7\x89\x88_\xe6\xa0\x91\xe5\xbd\xa2.txt'

Also teardown failed:
AttributeError: 'PlcSystemTestbench' object has no attribute 'dut_log'</status>
</test>
<status status="FAIL" endtime="20181122 21:49:30.951" starttime="20181122 21:49:17.068"></status>
</suite>
<status status="FAIL" endtime="20181122 21:49:30.951" starttime="20181122 21:49:17.038"></status>
</suite>
<statistics>
<total>
<stat fail="1" pass="0">Critical Tests</stat>
<stat fail="1" pass="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat fail="1" id="s1" name="Plc System Test" pass="0">Plc System Test</stat>
<stat fail="1" id="s1-s1" name="Tc.4" pass="0">Plc System Test.Tc.4</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
